src/main/java/sirius/pasta/tagliatelle/Tagliatelle.java

```diff
@@ -124,15 +125,29 @@
     }
 
     private List<TemplateExtension> loadExtensions(String target) {
-        return Sirius.getClasspath()
-                     .find(Pattern.compile("(default/|customizations/[^/]+/)?extensions/"
-                                           + Pattern.quote(target)
-                                           + "/.*.pasta"))
-                     .map(m -> m.group(0))
-                     .map(this::resolveToTemplateExtension)
-                     .filter(Objects::nonNull)
-                     .sorted()
-                     .collect(Collectors.toList());
+        List<TemplateExtension> extensionsForTarget = Sirius.getClasspath()
+                                                            .find(Pattern.compile("(customizations/[^/]+/)?extensions/"
+                                                                                  + Pattern.quote(target)
+                                                                                  + "/.*.pasta"))
```

Comment:
Extract pattern into constant so it is not always recompiled?

Reply:
this isn't constant as "target" is passed in - however the result of the method call is cached...